libs/blocks/marketo/marketo.css

```
@@ -378,6 +378,10 @@
     width: 47.7%;
   }
 
+  [class*="adobe-privacy"] fieldset.mktoFormCol.mktoVisible {
```

You don't want to scope it to the marketo block like the other styles?
```suggestion
  .marketo [class*="adobe-privacy"] fieldset.mktoFormCol.mktoVisible {
```